CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+### [v2.7 _(Jan 21, 2019)_](https://github.com/omise/omise-magento/releases/tag/v2.7)
+
+### 👾 Bug Fixes
+- Fix: Display correctly barcode from tesco lotus on checkout success page (PR [#180](https://github.com/omise/omise-magento/pull/180))
+- Fix: Order can't be completed when choosing other payment methods than Omise (PR [#179](https://github.com/omise/omise-magento/pull/179))
+
 ### [v2.6 _(Jan 15, 2019)_](https://github.com/omise/omise-magento/releases/tag/v2.6)
```

Comment:
@jacstn this one might be in Enhancement section
https://github.com/omise/omise-magento/pull/177